tests/integration/conftest.py

```diff
@@ -100,7 +100,7 @@ def inner(workspace_group_name=None, account_group_name=None):
         members = [user.id]
         ws_group = make_group(display_name=workspace_group_name, members=members, entitlements=["allow-cluster-create"])
         acc_group = make_acc_group(display_name=account_group_name, members=members)
-        return ws_group, acc_group
+        return ws_group, acc_group, user
```

Comment:
This change doesn't belong here - you see this by the amount of result ignores in tests. If you need a user for one of the tests - don't use this fixture there and write a lower-level code.

Reply:
will correct

Reply:
done